src/docs/product/performance/index.mdx

```diff
@@ -45,3 +45,18 @@ Below the graphs are a list of transactions that may be filtered down based on g
 ## Key Transaction View
 
 If you have transactions you frequently return to; you can mark each one as a key transaction in their corresponding [summary views](/product/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
+
+## Trends View
+
+Trends surfaces transactions that have had significant changes in their performance over time. This view can be found by toggling the tab in the upper right corner of the homepage.
+
+![trends Homepage](trends-homepage.png)
+
+Trending transactions are calculated first by filtering out transactions with large fluctuations in [throughput](/product/performance/metrics/#throughput-total-tpm-tps), then determining the baselines of the two halves of the duration, and calculating their percentage change. Regressions are any percentage changes greater than 0, while improvements are percentage changes less than 0.
+The default view for trends will only show high confidence trends, these are based on transactions that have a high [t-test](https://en.wikipedia.org/wiki/Welch%27s_t-test). High magnitude t-tests being used as a signal that there has been a significant change between periods.
```

Comment:
What about something like this?
```suggestion
The `Confidence` option allows you to filter trends based on High (trend is statistically significant) or Low (trend is not statistically significant) significance. By default, only high confidence, statistically significant results are shown. Sentry uses a [t-test](https://en.wikipedia.org/wiki/Welch%27s_t-test) to compare the before and after periods of the transaction and assign a confidence score. 
```

Reply:
Hmm what about this? The bits in the brackets seem a bit redundant to me
```suggestion
The `Confidence` option allows you to filter trends based on High or Low statistical significance. By default, only high confidence, statistically significant results are shown. Sentry uses a [t-test](https://en.wikipedia.org/wiki/Welch%27s_t-test) to compare the before and after periods of the transaction and assign a confidence score. 
```

Reply:
ya agreed that's better